rest_live/consumers.py

```diff
@@ -127,10 +127,17 @@ def receive_json(self, content: Dict[str, Any], **kwargs):
 
             # Check to make sure client has permissions to make this subscription.
             has_permission = True
-            for permission in view.get_permissions():
-                has_permission = has_permission and permission.has_permission(
-                    view.request, view
-                )
+                try:
+                    has_permission = has_permission and permission.has_permission(
```

Comment:
I think we probably still want the `for permission in view.get_permissions():` part?

Reply:
Definitely do, I pasted over one line too many.